cmd/cluster-olm-operator/main.go

```diff
@@ -124,9 +125,11 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		cc.EventRecorder.ForComponent("olm"),
 	)
 
+	operatorLoggingController := loglevel.NewClusterOperatorLoggingController(cl.OperatorClient, cc.EventRecorder.ForComponent("ClusterOperatorLoggingController"))
```

Comment:
Why does this feel like it needs to have "OLM" in it? (Like the other `ForComponents`?)

Reply:
Ah. I'll make it match. Good catch. My thinking at the time was "this is already scoped to the `openshift-cluster-olm-operator` namespace, but since the other components do it, we should be consistent.